### ISSUE TITLE
Get multiple profiles not working 

### ISSUE THREAD

**Issue by fourjr** [cr-api/cr-api#83]
I tried [this](http://api.cr-api.com/profile/2P0LYQ,92GQJLVC0) and have 

```json
{
"error":"Service Unavailable, but will be back shortly."
}
```

as a response.

The service works for other endpoints like [my profile](http://api.cr-api.com/profile/2P0LYQ) 

-Not hyperlinked multi profile: http://api.cr-api.com/profile/2P0LYQ,92GQJLVC0

**Comment by fourjr** [comment]
I get a 504 sometimes.

**Comment by selfish** [comment]
Oh this was a struggle, I hate this bug.
Resolved in 6.3
